scss/helpers/_icon-link.scss

```diff
@@ -0,0 +1,24 @@
+.icon-link {
+  display: inline-flex;
+  gap: .375rem;
+  align-items: center;
+  text-decoration-color: rgba(var(--#{$prefix}link-color-rgb), var(--#{$prefix}link-opacity, .5));
+  text-underline-offset: .5rem;
+  backface-visibility: hidden;
+
+  > .bi {
```

Comment:
**[Bootstrap Side ]** Missing `fill: currentColor` It comes from the doc css atm

Reply:
See https://github.com/twbs/bootstrap/pull/38130